test/ex_oauth2_provider/oauth2/authorization/strategy/code_test.exs

```diff
@@ -6,7 +6,7 @@ defmodule ExOauth2Provider.Authorization.CodeTest do
   alias Dummy.{OauthAccessGrants.OauthAccessGrant, Repo}
 
   @client_id                "Jf5rM8hQBc"
-  @valid_request            %{"client_id" => @client_id, "response_type" => "code", "scope" => "app:read app:write"}
+  @valid_request            %{"client_id" => @client_id, "response_type" => "code"}
```

Comment:
Can we keep this as it is, and instead just do `Map.delete(@valid_request, "scope")` in the "with no scope" test?

Reply:
@danschultzer : since scopes are an optional part of OAUTH i wanted to have some tests that did sanity check and make sure it would function.
I see no problem with keeping it. I could rename it to @valid_request_with_scope and make one for no scope.